src/nvim/tag.c

```diff
@@ -2783,7 +2783,11 @@ int get_tags(list_T *list, char_u *pat)
   if (ret == OK && num_matches > 0) {
     for (i = 0; i < num_matches; ++i) {
       int parse_result = parse_match(matches[i], &tp);
```

Comment:
If this is *not* used in Release build, then wouldn't it be better to wrap it in `#ifndef NDEBUG` to avoid the cost of `parse_match()`?